README.md

```diff
@@ -121,6 +175,46 @@ staticjinjaplus offers the following additional Jinja facilities.
 {{ dict(yes=True)|dictmerge(dict(no=False)) }} {# {"yes": True, "no": False} #}
 ```
 
+#### Markdown
+
+staticjinjaplus has generic support for working with Markdown-formatted files. Write your Markdown files (`.md`) as usual
+in the `TEMPLATES_DIR` directory: they will be rendered to HTML using the appropriate template partial, and saved using
+the source root pathname to the `OUTPUT_DIR` directory; i.e. `{TEMPLATES_DIR}/blog/awesome.md` will be rendered to
+`{OUTPUT_DIR}/blog/awesome.html`.
+
+staticjinjaplus do read metadata fields of Markdown files. Supported syntax is documented [here](https://python-markdown.github.io/extensions/meta_data/#syntax).
+The only reserved metadata field is `partial`, which tells staticjinjaplus which template partial (relative to the
+`TEMPLATES_DIR` directory) to use when rendering the Markdown file. See example below.
+
+It fallbacks to the `MARKDOWN_DEFAULT_PARTIAL` [configuration value](#configpy) if the field was not found. An error will
+be logged if staticjinjaplus can't determine which template partial to use, and rendering of the Markdown file will be
+canceled.
+
+This template partial will be given a Markdown-specific variable named `markdown` containing a dictionary which keys and
+values are detailed below.
+
+| Name                 | Type              | Description                                                                                                            |
+|----------------------|-------------------|------------------------------------------------------------------------------------------------------------------------|
+| `markdown.converted` | markupsafe.Markup | The resulting HTML, marked as safe to be rendered by Jinja. Metadata are of course not included in this output         |
+| `markdown.url`       | str               | The rendered Markdown template URL                                                                                     |
+| `markdown.meta`      | Dict[str, str]    | Metadata parsed from the source Markdown file. Multiline values have been concatenated using `\n` without extra spaces |
+
+Internal URLs must be written using the rendered version of the templates/assets/static files, i.e. URLs relative to what
+will be rendered in the `OUTPUT_DIR` directory. staticjinjaplus will not rewrite any URLs in any manner: you must provide
+the right ones by yourself. See example below.
+
```

Comment:
See also the `htmlcollect()` [Jinja global](#globals) which helps listing Markdown files, parsing their metadata, and more.